templates/music/music.html

```diff
@@ -17,7 +17,7 @@
             {% for album in albums %}
                 <div class="column">
                     <div class="artist-list-image img-list">
-                        {% include "object_preview.html" with object=album img=album.artwork.img %}
+                        {% include "object_preview_music.html" with object=album img=album.artwork.img %}
```

Comment:
The difference I wanted to have is for the artists lists to show the name by default, then hidden when hovered. For the music list it would show the artwork by default, then show the project and artist name when hovered.